components/CategoryImage/CategoryImage.tsx

```diff
@@ -0,0 +1,37 @@
+'use client';
+
+import type { Category, TranslatedCategory } from '@prezly/sdk';
+import Image from '@prezly/uploadcare-image';
+import classNames from 'classnames';
+
+import { type CardSize, getCardImageSizes } from './lib';
+
+import styles from './CategoryImage.module.scss';
+
+type Props = {
+    category: Category;
+    translatedCategory: TranslatedCategory;
+    size: CardSize;
+    className?: string;
+};
+
+export function CategoryImage({ category, translatedCategory, size, className }: Props) {
+    const { image } = category;
+
+    if (image) {
+        return (
+            <Image
+                imageDetails={image}
+                alt={translatedCategory.name}
+                layout="fill"
+                objectFit="cover"
+                containerClassName={classNames(styles.imageContainer, className)}
+                className={styles.image}
+                sizes={getCardImageSizes(size)}
+            />
+        );
+    }
+
+    // TODO add fallback image
+    return null;
```

Comment:
Are you planning on addressing this TODO? Right now if there's no image, the consumer code renders an empty link which creates some empty space.

Reply:
What should we do? We haven't settled on a fallback image. Maybe we can reuse the one for the stories?

Reply:
c1974274

Reply:
We also have a fallback for when there's no newsroom logo 🙂